types/react-virtualized/react-virtualized-tests.tsx

```diff
@@ -344,7 +344,7 @@ export class DynamicHeightList extends PureComponent<any> {
         return (
             <CellMeasurer cache={this._cache} columnIndex={0} key={key} rowIndex={index} parent={parent}>
                 {({ measure, registerChild }) => (
-                    <div ref={registerChild} className={classNames} style={style}>
+                    <div ref={registerChild as React.Ref<HTMLDivElement>} className={classNames} style={style}>
```

Comment:
Type-casting required due to https://github.com/microsoft/TypeScript/issues/30748